tensorly/decomposition/_cp.py

```diff
@@ -209,7 +209,7 @@ def error_calc(tensor, norm_tensor, weights, factors, sparsity, mask, mttkrp=Non
 
             # mttkrp and factor for the last mode. This is equivalent to the
             # inner product <tensor, factorization>
-            iprod = tl.sum(tl.sum(mttkrp * factors[-1], axis=0))
+            iprod = tl.sum(tl.sum(mttkrp * tl.conj(factors[-1]), axis=0))
```

Comment:
Why does it have to be the conjugate here?

Reply:
@yngvem had this insight, so he might be able to explain. I confirmed that the results are indeed wrong without this.